monitor/monitor.go

```diff
@@ -104,6 +94,36 @@ func RunMonitor(ctx context.Context, containerConfig build.ContainerConfig, relo
 					fmt.Fprint(stdout, helpMessage)
 				}
 			}
+			completer := func(d prompt.Document) []prompt.Suggest {
+				return []prompt.Suggest{}
+			}
+
+			prompt.New(exec, completer, prompt.OptionHistory([]string{
+				"reload", "rollback", "exit", "help",
```

Comment:
This sets the history at the beginning? I think we probably should just leave this empty (maybe a future enhancement to save the history to `~/.buildx/monitor_history` or similar?)